--- Makefile.orig      2005-07-09 00:08:47.000000000 +0200
+++ Makefile   2010-06-20 00:40:30.000000000 +0200
@@ -10,7 +10,7 @@
 DEFINES= -DRESOLVE_HOSTNAMES
 
 #options if you have a bind>=4.9.4 libresolv (or, maybe, glibc)
-LDLIBS=-lresolv
+#LDLIBS=-lresolv
 ADDLIB=
 
 #options for decnet
@@ -27,7 +27,7 @@
 
 LDLIBS += -L../lib -lnetlink -lutil
 
-SUBDIRS=lib ip tc misc netem
+SUBDIRS=lib ip tc
 
 LIBNETLINK=../lib/libnetlink.a ../lib/libutil.a

